### PR TITLE
Google Sheets integration

### PR DESCRIPTION
**Pull request by aleksix**
Adds support for parsing google sheets tables from the devised format.
This PR doesn't include the credentials file for security reasons.

### CODE REVIEW

File: livebook/parsers/stat.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Stat:
+
+    def __init__(self, alias, name, def_value, oaths):
+        self.alias = alias
+        self.name = name
+        self.def_value = def_value
+        self.oaths = oaths
```
Comment (cezarnik) [inline livebook/parsers/stat.py:7]: What is 'oaths' field supposed to do?

File: livebook/parsers/scene.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Scene:
+
+    def __init__(self, text, options):
+        self.text = text
+        self.options = options
```
Comment (cezarnik) [inline livebook/parsers/scene.py:5]: Do we insist that for each scene options should be distinct?

File: livebook/parsers/google_sheets/parser.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+# Get the sheet
+sheet = client.open_by_key(spreadsheet_id)
+# We'll nearly always need the meta sheet
+meta_sheet = sheet.worksheet("livebook_meta")
+
+# TODO: Good idea?
+'''
+Module variables
+'''
+stats_end = 1
```
Comment (cezarnik) [inline livebook/parsers/google_sheets/parser.py:25]: We can create a class Parser, and move the stats_end as a field. In such a way we can avoid using unnecessary column_val variable.

File: livebook/parsers/stat.py (hunk 1 above)
Comment (aleksix) [reply livebook/parsers/stat.py:7]: Just a list of oath descriptions.

File: livebook/parsers/scene.py (hunk 2 above)
Comment (aleksix) [reply livebook/parsers/scene.py:5]: No. No real reason to forbid non-distinct options.

File: livebook/parsers/google_sheets/parser.py (hunk 3 above)
Comment (aleksix) [reply livebook/parsers/google_sheets/parser.py:25]: The idea was that the parser is basically a Singleton, so we can use it as a module, without creating classes. Therefore, we can use module variables and everything else without caring too much.

**Pull request merged**